### PR TITLE
test(cleanup): cleanup in tests setup

### PR DESCRIPTION
**Pull request by araujobarret**
### Description

Some chores day in trying to reduce overload and organize a bit our jest setup file.
Changes:
- replaces reanimated outdated mock with the latest one 50607c696de1269a4eaad99abd3e7b26a009ba72
- removes unnecessary gesture handler mock d022e632cb39fe903c35eb2c28190cd1bbd117c3
- fixes a broken test 99debb303f04aebf6654f25b802ec031a5386388
- removes necessary cookie module mock 6c441f2d8b28911683c331b894b0adbdf32e65aa
- cleans up navigation and native modules beforeEach 0247f5aaae5aca5b937403e5ff52b819100fa464

PS: all tests are passing on CI and locally ✅

image

#noChangelog

### PR Checklist

- [ ] I have tested my changes on **iOS** and **Android**.
- [ ] I hid my changes behind a **[feature flag]**, or they don't need one.
- [ ] I have included **screenshots** or **videos**, or I have not changed the UI.
- [ ] I have added **tests**, or my changes don't require any.
- [ ] I added an **[app state migration]**, or my changes do not require one.
- [ ] I have documented any **follow-up work** that this PR will require, or it does not require any.
- [ ] I have added a **changelog entry** below, or my changes do not require one.

### To the reviewers 👀

- [ ] I would like **at least one** of the reviewers to **run** this PR on the simulator or device.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

-

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

Need help with something? Have a look at our [docs], or get in touch with us.

[app state migration]: ../blob/main/docs/adding_state_migrations.md
[feature flag]: ../blob/main/docs/developing_a_feature.md
[docs]: ../blob/main/docs/README.md

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 0612cb58636462f235f0c7286181f2cce326c2d6
</p>

File: src/setupJest.tsx [hunk 1]
```diff
@@ -559,24 +479,6 @@ jest.mock("app/NativeModules/LegacyNativeModules", () => ({
 
 Object.assign(NativeModules, getNativeModules())
 
-beforeEach(() => {
```
Comment (gkartalis) [inline src/setupJest.tsx:562]: 👏

File: src/setupJest.tsx [hunk 2]
```diff
@@ -309,46 +269,6 @@ jest.mock("react-native/Libraries/LayoutAnimation/LayoutAnimation", () => ({
   spring: jest.fn(),
 }))
 
-jest.mock("react-native-gesture-handler", () => {
```
Comment (gkartalis) [inline src/setupJest.tsx:312]: 👏

File: src/setupJest.tsx [hunk 3]
```diff
@@ -264,41 +258,7 @@ jest.mock("react-native-localize", () => ({
   },
 }))
 
-require("react-native-reanimated/src/reanimated2/jestUtils").setUpTests()
```
Comment (gkartalis) [inline src/setupJest.tsx:267]: 👏

**Review by egdbear — APPROVED** [review]

**Review by egdbear — APPROVED** [review]

**Review by egdbear — APPROVED** [review]

**Review by egdbear — APPROVED** [review]

**Review by egdbear — APPROVED** [review]

**Pull request merged**